pkg/inference/preset-inferences.go

```diff
@@ -105,6 +106,10 @@ func GetInferenceImageInfo(ctx context.Context, workspaceObj *kaitov1alpha1.Work
 		}
 		return imageName, imagePullSecretRefs
 	} else {
+		if strings.HasPrefix(workspaceObj.Name, "preset-adapter-") {
+			imageName := "aimodelsregistrytest.azurecr.io/kaito-falcon-7b-e2e:0.0.1"
+			return imageName, imagePullSecretRefs
+		} // Note: THIS IS ONLY FOR E2E TEST, TO LOAD THE INFERENCE API WITHOUT UPDALDING NEW IMAGE, WILL BE DELETED WHEN MERGE
```

Comment:
Ok so you made a custom image that includes you're print statement? Is that what this is

Reply:
Yes, this is an image with my changes of inference_api.py in it